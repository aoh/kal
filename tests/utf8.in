# flags: -t 0 
every day: λä.ä
