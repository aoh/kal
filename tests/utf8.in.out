1.1.1970, Thursday, week 1
 - λä.ä

2.1.1970, Friday, week 1
 - λä.ä

3.1.1970, Saturday, week 1
 - λä.ä
